Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+access or perhaps memory leakage.
+
+## Can I stake with encrypted wallet?
+
+Yes, but you have to unlock it first. It's okay to unlock it "for
+staking only".
+
+The console walletpassphrase command does what clicking the lock button
+does.
+
+To lock Type
+
+`walletlock`
+
+to unlock for a certain time type
+
+`walletpassphrase PASSWORD TIMEOUT`
+
+to unlock type
+
+`walletpassphrase PASSWORD`
+
+to unlock only for staking type
+
+`walletpassphrase PASSWORD TIMEOUT true`
+
+## How do you turn staking on and off?
+
+`walletpassphrase "your passphrase in double quotes" timeout true`
+
+where timeout is a number of seconds, e.g. 10000 and true is a toggle
+that unlocks the wallet for staking
+only.
+
+# BOINC
+
+## Choosing BOINC projects: In BOINC, is there still a defined whitelist of "projects" or can we use any project now that has Gridcoin as a Team Name?
+
+Running "list projects" in console will list all the whitelisted
+projects you could possibly contribute to that would help your Gridcoin
+Research
+wallet.
+
+## If it says "non participating project" does that mean that they aren't accepted for Gridcoin purposes?
+
+That means you aren't participating in those projects or the e-mail
+address you have in the conf file doesn't match the e-mail address used
+in your BOINC manager on that machine. For every project you join, you
+also have to join the Gridcoin team on that project. Any changes about
+e-mail address on a project or team membership on a project take around
+a day to
+propagate.
+
+## I use BOINC with one username but on a few machines - is it enough to have one wallet with the proper boinc\_project\_email address, or do I have to have a separate wallet on each machine?
+
+You only need one wallet for each BOINC account/CPID (Cross Project
+Identifier). You can use BAM (BOINC Account Manager) to combine more
+projects in one CPID. Here is a guide
+<http://grcnation.com/en/bam-boinc-account-manager/> . Your reward is
+based on the credit in your BOINC account/CPID. You can point as many
+machines and resources to a single account as you want and only need a
+single wallet to receive your reward. You do have to set "email=" in the
+config file to your BOINC/BAM email and must be running the wallet on a
+machine that has BOINC installed and is associated with all BOINC
+projects you would like to get credit for (they can be set to not
+retrieve new work) and each project must show at least some RAC on that
+machine (press "Update" in the BOINC software when a project which shows
+0 contribution is
+selected).
+
+## BOINC: For our accounts, we used to have to use our Gridcoin address as our usernames for those accounts. Is that the case still, and if so, should we change our address info over to the "new" client addresses? Like under BAM/BOINC Stats, our stats/info isn't searchable via e-mail address but is via username. Just wondering what we can/should use for going forward.

```

Comment:
I haven't heard of anyone asking this. I think we can probably get rid of this question